### PR TITLE
EOS public key address format.

### PR DESCRIPTION
**Pull request by krivenkoviktor1954**
Added support for EOS public key address format.

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import * as bech32 from 'bech32';
 import * as bs58check from 'bs58check';
 import * as cashaddr from 'cashaddrjs';
+import * as ecc from 'eosjs-ecc';
```
Comment (Arachnid) [inline src/index.ts]: Please name this something specific to EOS.

Comment (krivenkoviktor1954) [reply src/index.ts]: Changed to eos.

Comment (krivenkoviktor1954) [reply src/index.ts]: Scatter

scatter_eos

Comment (krivenkoviktor1954) [reply src/index.ts]: eos-Voter
voter

Comment (krivenkoviktor1954) [reply src/index.ts]: https://eosauthority.com/

eos_authority

**Pull request merged**